fe/fe-core/src/main/java/com/starrocks/qe/SessionVariableConstants.java

```diff
@@ -23,4 +23,6 @@ private SessionVariableConstants() {}
     public static final String FORCE_STREAMING = "force_streaming";
 
     public static final String FORCE_PREAGGREGATION = "force_preaggregation";
+
+    public static final String LIMITED = "limited";
```

Comment:
force_mem_limited_agg?

Reply:
Consider limiting based on num rows later. So I don't want to bind it to memory or force for now.